index.html

```diff
@@ -9,13 +9,16 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body>
-    <header class="nav">
```

Comment:
use semantic web :nav tage

Reply:
- [ ] Please always includes an explanation, a "why", for each requested change. For example, this is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
This request lacks a "why": *Why don't you use an article tag here instead of a div?*

- [ ] Please make all your comments clear and easy to understand. For Example, this is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
This is a vague comment: *The README is not descriptive, why don't you make it better?*

- [ ] Please phrase all your comments politely, as suggestions instead of orders. For example, This is a polite suggestion: *"I think it would be better to use semantic tags"*😊
This reads as an order: *"Use semantic tags"*

- [Optional] Consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.😁🙃